docs/core/tools/project-json-to-csproj.md

```diff
@@ -265,23 +300,6 @@ dependency version of a project reference.
 </ItemGroup>
 ```
 
-#### type: platform
-```json
-{
-  "dependencies": {
-    "Microsoft.NETCore.App": {
-      "version": "1.1.0",
-      "type": "platform"
```

Comment:
Ok, so how about I add that part back without mentioning the part about the implicit reference to metapackage? I think that was the part that he thought it was inaccurate because the fact that type: platform not supported had nothing to do with the implicit references.